weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -351,7 +351,12 @@ const MainPeekingLayout: FC = () => {
 
       <Box
         sx={{
-          borderLeft: '1px solid rgba(0, 0, 0, 0.12)',
+          borderLeft:
+            flexDirection === 'row' ? '1px solid rgba(0, 0, 0, 0.12)' : 'none',
```

Comment:
Just noting that we should start working towards using named colors from the official palette.

Reply:
I need to do this moving forward - you are correct. I fixed this case, but probably need to write a script to look through all of our weave js files and convert to the closest common color